src/qrcore.h

```diff
@@ -97,6 +112,8 @@ public slots:
     QString default_arch;
     QString default_cpu;
     int default_bits;
+
+    RCore *core_;
```

Comment:
It's a private member and should never be used, but I'm fine with naming it differently. My personal preference is `mMember` everywhere (I hate typing `_`) but it's not enforced anywhere currently anyway so we might as well call it `c_o_r_e_`